concreate/schema/common.yaml

```diff
@@ -66,3 +66,9 @@ map:
       install:
         seq:
           - {type: str}
+  osbs:
```

Comment:
Shouldn't this be part of the image.yaml only? I don't think we will be building modules as images.

Reply:
what about overrides?

Reply:
In overrides too. For such case, where we override it - the "base" image.yaml should not contain the `osbs` key even.

Reply:
I think it even make sense for module - we can have centos/fedora/rhel flavor of module which describes only where image is builded instead of overrides.